internal/hcl/funcs/marks.go

```diff
@@ -12,7 +12,7 @@ import (
 type valueMark string
 
 func (m valueMark) GoString() string {
-	return "marks." + strings.Title(string(m))
+	return "marks." + cases.Title(language.English).String(string(m))
```

Comment:
> SA1019: strings.Title has been deprecated since Go 1.18 and an alternative has been available since Go 1.0: The rule Title uses for word boundaries does not handle Unicode punct
uation properly. Use golang.org/x/text/cases instead. (staticcheck)